package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
We have a separate publish process that we use actually -- can you revert this change? I'll publish `1.2.0` once we merge this change.